<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_azimport</class>
 <widget class="QDialog" name="Dialog_azimport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>671</width>
     <height>441</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_import">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>import</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_von">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_mitarb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_bis">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_von">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>von:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_bis">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>bis:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
